predicators/envs/sokoban.py

```diff
@@ -364,3 +361,14 @@ def _is_static(cls, obj: Object, state: State) -> bool:
     @classmethod
     def _is_dynamic(cls, obj: Object, state: State) -> bool:
         return not cls._is_static(obj, state)
+
+    @staticmethod
+    def _get_task_seed(train_or_test: str, task_idx: int) -> int:
+        assert task_idx < CFG.test_env_seed_offset
+        entropy = CFG.seed
```

Comment:
Explain what this and the following lines are doing, so I don't have to refer to np.random.SeedSequence documentation.